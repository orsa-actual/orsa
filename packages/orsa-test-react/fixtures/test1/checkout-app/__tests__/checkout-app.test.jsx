/* eslint no-undef: 0 */
import React from 'react';
import renderer from 'react-test-renderer';

import CheckoutApp from '../src/checkout-app';

test('CheckoutApp outputs something', () => {
  const component = renderer.create(
    <CheckoutApp />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
