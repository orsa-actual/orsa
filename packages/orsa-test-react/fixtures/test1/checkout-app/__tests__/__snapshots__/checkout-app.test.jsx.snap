// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutApp outputs something 1`] = `
<div
  className="panel "
>
  <h1>
    Checkout
  </h1>
  <div>
    You need to login before you can checkout.
    <div
      className="panel "
    >
      <h1>
        Login
      </h1>
      <div>
        <form>
          <div
            className="form-field"
          >
            <div
              className="field-title"
            >
              Username
            </div>
            <input
              disabled={false}
              type="text"
              value=""
            />
          </div>
          <div
            className="form-field"
          >
            <div
              className="field-title"
            >
              Password
            </div>
            <input
              disabled={false}
              type="password"
              value=""
            />
          </div>
          <div>
            <button
              disabled={false}
            >
              foo!
            </button>
          </div>
        </form>
      </div>
    </div>
    <button
      disabled={true}
    >
      Checkout
    </button>
  </div>
</div>
`;
